QOL improvements and doc updates (#1318)
huggingface/peft: docs/source/developer_guides/lora.md

# delete adapter
model.delete_adapter("dpo")
```

> ## Utilities

Reviewer: ```suggestion
## Finding linear layers
```

> Q-LoRA paper outlines that best performance is achieved by targetting all the Linear layers of the model instead of only targetting the query and value layers. To find all the linear layers to target, we provide a utility function `get_linear_layer_names`. Below is the sample usage of it:

Reviewer: Is targeting all the linear layers for best performance specific to QLoRA, or is it also true for LoRA in general? If it's the latter, maybe we can remove mentioning the QLoRA paper.

```suggestion
In the QLoRA paper, the best performance is achieved by targeting all the linear layers of the model instead of only targeting the query and value layers. To find all the linear layers to target, use the [`~utils.get_linear_layer_names`] function:
```
